pubspec.yaml

```diff
@@ -5,6 +5,7 @@ author: jaumard <jimmy@kiwi-bop.com>
 homepage: https://github.com/kiwi-bop/flutter_crashlytics
 
 dependencies:
+  stack_trace: ^1.9.3
```

Comment:
Hum I didn't know about this library ! For sure if it work with this one it's better to migrate to this one :)